src/provider/cloud-element-templates/ElementTemplates.js

```diff
@@ -3,17 +3,24 @@ import {
   getTemplateVersion
 } from './Helper';
 
+import {
+  removeTemplate as removeTemplateHelper
```

Comment:
We'd re-use this via re-using the super-class implementation (cf. `DefaultElementTemplate`.